src/app/home/panel/navbar/navbar.component.ts

```diff
@@ -7,9 +8,25 @@ import { Component, OnInit } from '@angular/core';
 })
 export class NavbarComponent implements OnInit {
 
-  constructor() { }
+  constructor(private router: Router) {
+  }
 
   ngOnInit(): void {
   }
 
+  goToDashboard() {
+    this.router.navigate(['/dashboard']);
+  }
+
+  goToWallet() {
+    this.router.navigate(['/wallet']);
+  }
+
+  goToTransactions() {
+    this.router.navigate(['/transactions']);
+  }
+
+  goToBudgets() {
+    this.router.navigate(['/budgets']);
```

Comment:
There is some repeatability here, maybe you should try to create a generic method:
```
goTo(path: string) {
    this.router.navigate([path]);
}
```
And then use it on HTML like this: `(click)="goTo('/budgets')"`